one-pager-maker/src/pages/Entry.tsx

```diff
@@ -0,0 +1,28 @@
+import Markdown from "react-markdown";
+import { useParams } from "react-router-dom";
+import { documentApi } from "../api/documentApi.ts";
+
+function Entry() {
+  const { uid, documentId } = useParams<{
+    uid: string;
+    documentId: string;
+  }>();
+  if (uid === undefined || documentId === undefined) {
+    return <main>Route setting is wrong</main>;
+  }
+  const documentResult = documentApi.useGetDocumentQuery({ uid, documentId });
+  const document = documentResult.data?.value;
+  return (
+    <>
+      <div>
```

Comment:
nits: These `<>` and `<div>` might be unnecessary?\

Reply:
delete `<>`